src/main/java/org/isf/visits/service/VisitsIoOperationRepository.java

```diff
@@ -21,7 +21,7 @@
 	List<Visit> findAllWhereWardByOrderPatientAndDateAsc(@Param("wardId") String wardId);
 
 	@Query(value = "SELECT * FROM VISITS WHERE VST_WARD_ID = :ward ORDER BY VST_DATE", nativeQuery= true)
-	public List<Visit> findAllWhereWardByOrderDateAsc(@Param("ward") String ward);
+    List<Visit> findAllWhereWardByOrderDateAsc(@Param("ward") String ward);
```

Comment:
could please you use the https://github.com/informatici/openhospital-core/blob/develop/.editorconfig included in the project?

Reply:
I am using .editorconfig and Intellij and I am not seeing the indentation issue that is showing up here in the diffs.  I am assuming it has to do with "tab size"; that is why I much prefer only using spaces. :)  I'll continue to look into this.  Sorry for the problem.

Reply:
As I suspected some files seem to be a mix of space indentation and tab indentation (at least this one is).  In the following screenshot lines 15 through 18 use spaces and lines 20 through 27 use tab indentation. 
![image](https://user-images.githubusercontent.com/1105445/88064145-e3e0f780-cb38-11ea-8104-ca9284bb7683.png)

FWIW, the lines with spaces were added in this commit: 85c8f64e28ad645c9dc9b892749fcb16a2b6d32 on March 31, 2020

It appears that another standardizing pass like this [PR](https://github.com/informatici/openhospital-core/pull/50/files) is in order again.

Reply:
I just ran a single command line based command that converted all spaces to tabs in all java files in the `develop` branch and 174 files were changed in `main` and 129 files were changed in `test`.